cpp/log/leveldb_db-inl.h

```diff
@@ -47,18 +47,37 @@ std::unique_ptr<const leveldb::FilterPolicy> BuildFilterPolicy() {
 #endif
 
 
-std::string FormatSequenceNumber(const int64_t seq) {
-  return std::to_string(seq);
+std::string IndexToKey(int64_t index) {
+  CHECK_EQ(8, sizeof(index));
+  const char nibble[] = "0123456789abcdef";
+  char index_str[sizeof(index) * 2];
```

Comment:
Could avoid the string c'tor copy below with by having a string here and `reserve()`ing the length, and just add the chars in the loop below.